bridge/engine/ActiveMediaList.cpp

```diff
@@ -288,8 +288,13 @@ size_t ActiveMediaList::rankSpeakers(float& currentDominantSpeakerScore)
     for (auto& participantLevelEntry : _audioParticipants)
     {
         const auto& participantLevels = participantLevelEntry.second;
-        const float participantScore = participantLevels._maxRecentLevel - participantLevels._noiseLevel;
-        // std::max(0.0f, participantLevels._maxRecentLevel - participantLevels._noiseLevel);
+        if (participantLevels._maxRecentLevel == 0)
+        {
+            continue;
+        }
+
+        const float participantScore =
+            std::max(0.0f, participantLevels._maxRecentLevel - participantLevels._noiseLevel);
```

Comment:
Don't we want it like this:

```suggestion
      const float participantScore =
            std::max(0.0f, participantLevels._maxRecentLevel - participantLevels._noiseLevel);
            
        if (participantScore == 0)
        {
            continue;
        }
```